FAO Publications.js

```diff
@@ -281,28 +288,83 @@ function doWeb(doc, url) {
 var testCases = [
 	{
 		"type": "web",
-		"url": "http://www.fao.org/documents/card/en/c/204a6894-b554-472e-9a9b-d84404dfcb9e/",
+		"url": "http://www.fao.org/documents/card/zh/c/d8a82e37-695b-4eca-b6f9-53c12758eb8e/",
```

Comment:
Are the automatic test in Scaffold working for you?

Reply:
Not quite... When I select an existing test (e.g. populated from the js file) to run or update, Scaffold reports error: "Translation failed: TypeError: ZU.xpath(...)[0] is undefined". So I need to use "New Web" every time a test is needed.

Reply:
Sorry. This was an issue when this translator was first created (https://github.com/zotero/translators/pull/1409 ). I've added a comment before the testing cases.